src/legacy/core_plugins/kibana/public/visualize/wizard/new_vis_modal.tsx

```diff
@@ -24,7 +24,7 @@ import { i18n } from '@kbn/i18n';
 
 import chrome from 'ui/chrome';
 import { VisType } from 'ui/vis';
-import { VisTypeAlias } from 'plugins/visualizations';
+import { VisTypeAlias } from 'src/legacy/core_plugins/visualizations/public/np_ready/public';
```

Comment:
As #44306 said to create `np_ready` folder, I did so. But I also thought that it wasn't necessary because `visualizations` plugin doesn't have any legacy code right now. 

I'll revert it back to the first non-`np_raedy` version.

Reply:
After investigating a bit, I found out that `np_ready` cannot be removed. 

As `visualizations` lives in `legacy/core_plugins` it needs to have `package.json` and `index.ts` files for legacy plugin initialization. 

When we remove them, we should merge legacy `index.ts` and np `index.ts`. 

It seems that `np_ready` is necessary evil for now.

p.s. I've changed all paths to relative in the below.